flocker/node/_docker.py

```diff
@@ -431,10 +431,16 @@ def _create():
                 port_bindings=port_bindings,
                 restart_policy=restart_policy_dict,
             )
+            # We're likely to get e.g. pvector, so make sure we're passing
+            # in something JSON serializable:
```

Comment:
Not just likely, but this is what the docstring says that this `ivar` will be.

:question: Explain why this must be something JSON serialisable - looking at `docker-py` it is because a JSON formatted `str` is used as a config file.